sklearn/utils/seq_dataset.pxd.tp

```diff
@@ -0,0 +1,85 @@
+{{py:
+
+"""
+Dataset abstractions for sequential data access.
+
+Template file for easily generate fused types consistent code using Tempita
+(https://github.com/cython/cython/blob/master/Cython/Tempita/_tempita.py).
+
+Generated file: seq_dataset.pxd
+
+Each class is duplicated for all dtypes (float and double). The keywords
+between double braces are substituted in setup.py.
+"""
+
+# name, c_type
+dtypes = [('', 'double'),
+          ('32', 'float')]
```

Comment:
it is much more clear if we use: 
```Python
 dtypes = [('64', 'double'),
￼            ('32', 'float')]
```
I found an error complaining about `SequentialDataset` and I had a hard time to figure out that it was referring to the 64bits version. @Henley13 do you remember why we choose `''` over `'64'`?